tests/integration/test.events.js

```diff
@@ -47,7 +47,7 @@ adapters.forEach(function (adapter) {
     });
 
     it('emit creation event', function (done) {
-      var db = new PouchDB(dbs.name).on('created', function (newDB) {
+      var db = new PouchDB(dbs.name).once('created', function (newDB) {
```

Comment:
This was a preexisting racy test, which was revealed to be racy because now IDB takes much longer to start up.